src/components/network-chart/NetworkChartRow.js

```diff
@@ -493,7 +502,8 @@ export class NetworkChartRow extends React.PureComponent<
           width={width}
           timeRange={timeRange}
         />
-        {shouldDisplayTooltips() && this.state.hovered ? (
+        {(shouldDisplayTooltips() && this.state.hovered) ||
+        (shouldDisplayTooltips() && this.props.isSelected) ? (
```

Comment:
I think we should remove this change in this PR. We can look at it in PR3, but for now this looks broken.